src/order/retrieve-order.ts

```diff
@@ -0,0 +1,68 @@
+import { fetch } from "undici";
```

Comment:
I like the idea of renaming this file to `get-order.ts`, now that the function has been renamed.